sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -476,6 +477,10 @@ of these enumerators, and it provides a brief description of their meanings.
 |2
 |GCN GFX10.3 (RDNA 2) architecture.
 
+|`amd_gpu_gfx1034`
+|3
```

Comment:
You are saying here that this new enum value is added in version 3 of the this extension.  Therefore, you need to also do the following:

* Add a new row to the "Feature test macro" section above in this document.
* Change the pre-define of `SYCL_EXT_ONEAPI_DEVICE_ARCHITECTURE` in the header file to set it to `3`.

Reply:
@gmlueck gfx1034 is not a new architecture, and could have been added with other amd/nvidia GPUs. Would you be against having it as  one of the architectures for revision 2, instead of adding a revision for this architecture?

Reply:
The version identifies the version of the extension's API, and we did not think to add this enum in version 2.  Adding it now changes the API.

Another option is to remove the API versioning entirely.  We require API versioning for "supported" extensions, but not for "experimental" ones.  If we want to go this route, then:

* Change the "Feature test macro" section of the document to follow the wording in the [template](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc#feature-test-macro) (see proposed wording for "experimental extension").
* In the table defining the enumerators, keep the column "Added in version", but change all the numeric values to "-" (dash).  Keeping the column will make it easier to add the versioning back when the extension becomes "supported".
* Change the pre-define of the `SYCL_EXT_ONEAPI_DEVICE_ARCHITECTURE` macro to just `#define SYCL_EXT_ONEAPI_DEVICE_ARCHITECTURE` (with no value).

Reply:
@gmlueck thanks.

Reply:
@gmlueck Done.